### ISSUE TITLE
3D Pose estimation on custom data

### ISSUE THREAD

**Issue by userVl** [open-mmlab/mmpose#1262]
Hi, I'm trying to lift 2D pose to 3D using custom model and custom data.  My 2D pose estimation model outputs the 17 COCO keypoints in [0,1] interval. My model is in ONNX format so I'm also converting simple baseline 3D to ONNX format and need custom code for data processing. I convert the COCO keypoints to H36M format, but further from this I see that the demo uses parameters to normalize the 2D coordinates with values that I feel are particular to the H36M resolution and annotations. How should I process my 2D keypoints such that the converted ONNX model can correctly predict from them?

Thanks!

**Comment by ly015** [comment]
The normalization is for alleviating the gap between the training data and the real scenarios. Its parameters are obtained from the stats of h36m training data which is used for training our models. If you plan to use our pretrained 2D-to-3D lifting models on h36m, these parameters should work just fine.

Please also note that the normalization is no more than an engineering trick that we found may help improve the demo in some cases.  You may need to rethink if it is suitable for your projects.
